src/code/z_viszbuf.c

```diff
@@ -0,0 +1,98 @@
+/**
+ * @file z_viszbuf.c
+ * Description: Visualise the z-buffer (AKA depth buffer), using cycling RGBA or a single fading color.
+ *
+ * useRgba:
+ * - false will produce a monotonic fade from primColor to envColor as depth increases
+ * - true produces vibrant almost-periodic-looking bands.
+ *
+ * In both cases this occurs because of the format the depth information takes: it is 18-bit, and is a positive
+ * floating-point number with
+ *     bbb      mmmmmmmmmmm dd|dd
+ *     exponent mantissa    dz value (only first 16 bits visible to CPU)
+ *
+ * IA:
+ *     bbbmmmmm mmmmmmdd
+ *     iiiiiiii aaaaaaaa
+ * Since positive floating-point numbers have the same ordering as their binary/hex representation, increasing
```

Comment:
```suggestion
 * Since floating-point numbers following this unsigned format have the same ordering as their binary/hex representation, increasing
```
my main point here being that you may very well come up with an unorthodox floating point format where the binary representation is inconvenient, so this technically isn't a statement that is in general true